pages/common/ssh.md

```diff
@@ -15,15 +15,15 @@
 
 `ssh {{username}}@{{remote_host}} -p {{2222}}`
 
-- Run a command on a remote server:
+- Run a command on a remote server (use `-t` to force a [t]ty allocation allowing interaction with the remote command):
```

Comment:
Hmmm. I'm on the fence about this one, as it seems like it's essentially a second example.

Reply:
I also think this should be either a separate example or be removed